<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeagueScreen</class>
 <widget class="QWidget" name="LeagueScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>700</height>
    </rect>
   </property>
   <widget class="QWidget" name="LeagueMenu">
    <widget class="QPushButton" name="playLeagueMatch">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>600</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Play Next Match</string>
     </property>
    </widget>
    <widget class="QLabel" name="nextMatches">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Previous Results</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Next Round Matches</string>
     </property>
    </widget>
    <widget class="QPushButton" name="simRound">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>650</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Sim Round</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="standings">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="previousMatches">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="MatchScreen" name="MatchWidget"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatchScreen</class>
   <extends>QWidget</extends>
   <header>MatchScreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
